app/controllers/lti_controller.rb

```diff
@@ -0,0 +1,52 @@
+class LtiController < ApplicationController
+  skip_verify_authorized
+
+  before_action :authenticate, :check_course_switch, :check_record,
+                except: [:get_config]
+
+  def get_config
+    # See https://canvas.instructure.com/doc/api/file.lti_dev_key_config.html
+    # for example configuration and descriptions of fields
+    config = {
+      title: 'Markus',
+      description: 'Markus',
+      oidc_initiation_url: "#{root_url}lti/launch",
+      target_link_uri: "#{root_url}login",
+      scopes: [
+        'https://purl.imsglobal.org/spec/lti-ags/scope/lineitem',
```

Comment:
let's keep things minimal and leave `scopes` empty for now, until we make the PRs to add in the relevant functionality